### PR TITLE
feat: filename hyperlink for console text output

### PR DESCRIPTION
**Pull request by brenoepics**
image
image
image

This feature helps to open files faster when we have many alerts to check.

### CODE REVIEW

**Comment by rrd108** [comment]
will this work on windows also?

**Review by rrd108 — CHANGES_REQUESTED** [review]
please remove dist/vue-mess-detector.es.js from the PR.

I will regenerate it on release.

**Comment by brenoepics** [comment]
> will this work on windows also?

Yes

**Comment by rrd108** [comment]
There is some problem with this PR as it shows me 0 files changed in it.

Can you please check it on your side?

**Comment by brenoepics** [comment]
Done

File: src/cli.ts [hunk 1]
```diff
@@ -106,9 +107,17 @@ getProjectRoot(pathArg || './src').then(async (projectRoot) => {
             })
 
             for (const group in result.reportOutput) {
-              log(`\n- <text_info> ${group}</text_info>`)
+              let groupOutput = group
+              if (argv.group === 'file') {
+                groupOutput = `\u001B]8;;file://${path.resolve(group)}\u001B\\${group}\u001B]8;;\u001B\\`
```
Comment (rrd108) [inline src/cli.ts:112]: What are these escape characters? Can you move them to the constants? They look scary 😱

**Review by rrd108 — CHANGES_REQUESTED** [review]

File: src/cli.ts (hunk 1 above)
Comment (brenoepics) [reply src/cli.ts:112]: Sure, and they are a control sequence used by terminal emulators to create clickable hyperlinks.

reference:
https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda

**Pull request merged**